src/app/containers/Include/index.jsx

```diff
@@ -18,13 +20,35 @@ const IncludeContainer = ({ html, type }) => {
     return null;
   }
 
+  const requireIncludeTypes = ['vj', 'idt1'];
+
+  const paths = `{
+    'jquery-1':
+      'https://static.bbc.co.uk/frameworks/jquery/0.4.1/sharedmodules/jquery-1.7.2',
+    'istats-1':
+      'https://nav.files.bbci.co.uk/orbit-webmodules/0.0.2-448.45dcd56/istats/istats-1',
+  }`;
+
+  const configureAdditionalScripts = `require.config({ paths:${paths} });`;
+
   return (
-    <GridItemConstrainedMedium>
-      <div
-        suppressHydrationWarning
-        dangerouslySetInnerHTML={{ __html: html }}
-      />
-    </GridItemConstrainedMedium>
+    <>
+      {requireIncludeTypes.includes(type) ? (
+        <Helmet>
+          <script
+            type="text/javascript"
+            src="https://news.files.bbci.co.uk/include/vjassets/js/vendor/require-2.1.20b.min.js"
+          />
```

Comment:
>Does this have to go into the head, or could it be injected immediately before the include? If the latter I was wondering if that might reduce the delay to start render 

We could add into the body, the trouble we be de-dupeing the script across multiple includes, I imagine this would be pretty complex, I'm not sure it's worth the effort imo.

> If there are multiple includes, will they each result in this being added?

As above, `Helmet`'s de-dupeing takes care of this problem, I'll add a test to demonstrate this as I don't think we have a test to prove this.

Reply:
Added a test to prove the above: 466ae67a76eea34f47631a926d3c6c8d79a848f2

Reply:
just ran the branch locally, from James's questions above, I can see that
1. the resulting script gets placed into the head
2. for multiple includes, it resolves to single require, jquery and istats scripts being loaded